packages/ui5-config/src/defaults.ts

```diff
@@ -2,11 +2,11 @@ export const enum UI5_DEFAULT {
     DEFAULT_UI5_VERSION = '',
     DEFAULT_LOCAL_UI5_VERSION = '1.95.0',
     MIN_UI5_VERSION = '1.60.0',
-    MIN_LOCAL_SAPUI5_VERSION = '1.76.0',
+    MIN_LOCAL_SAPUI5_VERSION = '1.76.0', // eslint-disable-line @typescript-eslint/no-duplicate-enum-values
     MIN_LOCAL_OPENUI5_VERSION = '1.52.5',
     SAPUI5_CDN = 'https://ui5.sap.com',
     OPENUI5_CDN = 'https://sdk.openui5.org',
-    TYPES_VERSION_SINCE = '1.76.0',
+    TYPES_VERSION_SINCE = '1.76.0', // eslint-disable-line @typescript-eslint/no-duplicate-enum-values
```

Comment:
Why are the two eslint configuration comments needed?

Reply:
Reduced to 1 in https://github.com/SAP/open-ux-tools/pull/1641/commits/fc71c65823caa490be5c62215e2882d96bcce10a